fitparser/src/error.rs

```diff
@@ -4,7 +4,7 @@ use std::io;
 use std::{error, fmt};
 
 /// The result of a deserialization operation.
-pub type Result<T> = ::std::result::Result<T, Error>;
+pub type Result<T, E = Error> = ::std::result::Result<T, E>;
```

Comment:
What does this change do? I don't quite understand the difference but my rust is a bit 'rusty'.

Reply:
In both cases, `Result<T>` resolves to `Result<T, Error>`.  But if I want to use a different error type, e. g. `TryFromRecordError` in the `TryFrom` implementation, I cannot use the original typedef because it always uses `Error`.  So I would have to use `::std::result::Result<T, TryFromRecordError>` or `use std::result::Result as StdResult` and then `StdResult<T, TryFromRecordError>`.  With the changed typedef, `Error` is just a default that I can overwrite, so I can write `Result<T, TryFromRecordError>`.